### DISCUSSION TITLE
WIndows YourPhone

### DISCUSSION THREAD

**Discussion by oid2000** [AndrewRathbun/DFIRArtifactMuseum#23] [Ideas]
Hi there,
Could you please provide a sample databases of the Windows 10's Your Phone application? The databases contain some Android-specific info and can be found here:
%LocalAppData%\Local\Packages\Microsoft.YourPhone_8wekyb3d8bbwe\LocalCache\Indexed\#GUID#\System\Database\

**Comment by AndrewRathbun** [comment]
Can you make an issue for this as an artifact request?
